planning/planning_validator/src/planning_validator.cpp

```diff
@@ -429,10 +449,11 @@ bool PlanningValidator::checkValidDistanceDeviation(const Trajectory & trajector
 
 bool PlanningValidator::isAllValid(const PlanningValidatorStatus & s)
```

Comment:
```suggestion
bool PlanningValidator::isAllValid(const PlanningValidatorStatus & s) const
```

Reply:
this is not modified part so no need to modify, but we can add `const` some functions in this class

Reply:
Done in https://github.com/autowarefoundation/autoware.universe/pull/6126/commits/992f7c123df3d8d5a019fab11c5d5e32f9f4237a